apierror.h

```diff
@@ -68,6 +70,6 @@
 /*! \brief Helper method to get a string representation of an API error code
  * @param[in] error The API error code
  * @returns A string representation of the error code */
-const char *janus_get_api_error(int error);
+shared const char *janus_get_api_error(int error);
```

Comment:
@lminiero it's defined here: https://github.com/meetecho/janus-gateway/pull/597/files#diff-b8c9b43cc2d006d6edf7b73b4a54fe10R22 but I have to admit it also threw me off the first time. Usuallt this is called something like JANUS_API or JANUS_EXTERN. IMHO something like that reads better.